src/components-examples/cdk/accordion/cdk-accordion-overview/cdk-accordion-overview-example.html

```diff
@@ -1,31 +1,31 @@
 <cdk-accordion class="example-accordion">
   @for (item of items; track item; let index = $index) {
-    <cdk-accordion-item
-
-      #accordionItem="cdkAccordionItem"
-      class="example-accordion-item"
-      role="button"
+  <cdk-accordion-item #accordionItem="cdkAccordionItem" class="example-accordion-item">
+    <button
+      class="example-accordion-item-header"
+      (click)="accordionItem.toggle()"
       tabindex="0"
       [attr.id]="'accordion-header-' + index"
       [attr.aria-expanded]="accordionItem.expanded"
       [attr.aria-controls]="'accordion-body-' + index">
-      <div class="example-accordion-item-header" (click)="accordionItem.toggle()">
-        {{ item }}
-        <span class="example-accordion-item-description">
-          Click to {{ accordionItem.expanded ? 'close' : 'open' }}
-        </span>
-      </div>
-      <div
-        class="example-accordion-item-body"
-        role="region"
-        [style.display]="accordionItem.expanded ? '' : 'none'"
-        [attr.id]="'accordion-body-' + index"
-        [attr.aria-labelledby]="'accordion-header-' + index">
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Perferendis
-        excepturi incidunt ipsum deleniti labore, tempore non nam doloribus blanditiis
-        veritatis illo autem iure aliquid ullam rem tenetur deserunt velit culpa?
-      </div>
-    </cdk-accordion-item>
+      {{ item }}
+      <span class="example-accordion-item-description">
+        Click to {{ accordionItem.expanded ? 'close' : 'open' }}
+      </span>
+    </button>
+    @if(accordionItem.expanded){
```

Comment:
nit: the formatting is off here

Reply:
I fix formatting problem in if statement. @crisbeto